README.md

```diff
@@ -37,7 +37,7 @@ You will need Ruby installed.
     If this fails with error messages that mention `pg` or PostgreSQL, then
     try instead:
 
-        bundle install --without-production
+        bundle install --without=production
```

Comment:
I found this was wrong when I was following the instructions like a complete Ruby noobie!

Reply:
Good catch, thanks! :laughing: